src/client/app/services/exportData.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import moment from 'moment';
+
+/**
+ * Function to converts the compressed meter data into a CSV formatted string.
+ * @param items The compressed meter data.
+ * @returns output A string containing the CSV formatted compressed meter data.
+ */
+
+function convertToCSV(items) {
+	let csvOutput = 'id:,readings:,timestamp:\n';
```

Comment:
You should change `timestamp` to `start_timestamp` and `end_timestamp`. You should also remove the colons in the labels.